src/backend/linux_raw/param/auxv.rs

```diff
@@ -121,14 +123,31 @@ pub(in super::super) fn sysinfo_ehdr() -> *const Elf_Ehdr {
     ehdr
 }
 
+#[cfg(feature = "runtime")]
+#[inline]
+pub(crate) fn entry() -> usize {
+    let mut entry = ENTRY.load(Relaxed);
+
+    if entry == 0 {
+        init_auxv();
+        entry = ENTRY.load(Relaxed);
+    }
+
+    entry
+}
```

Comment:
Shouldn't this be a pointer for provenance reasons? I'm not familiar with Linux program startup.1